v2_papers/2009/kind_histogram.json

```diff
@@ -0,0 +1,179 @@
+{
+  "version": "v2.1",
+  "reference": {
+    "title": "Histogram-Based Traffic Anomaly Detection",
+    "authors": [
+      "Kind, Andreas",
+      "Stoecklin, Marc Ph.",
+      "Dimitropoulos, Xenofontas"
+    ],
+    "publication_name": "IEEE Transactions on Network Service Management",
+    "publication_type": "peer_reviewed_journal",
+    "year": 2009,
+    "organization_publishers": [
+      "ieee"
+    ],
+    "pages_number_of": 12,
+    "bibtex": {
+      "type": "article",
+      "volume": "6",
+      "issue": "2",
+      "pages": "110--121"
+    },
+    "access_open": false,
+    "curated_by": "Meghdouri, f.",
+    "curated_last_revision": "11-04-2018",
+    "curated_revision_number": 2
+  },
+  "data": {
+    "datasets": [
+      {
+        "name": "campus of IBM Research",
+        "types": [],
+        "generation_year": 2004,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured"
+      },
+      {
+        "name": "data center",
+        "types": [],
+        "generation_year": 2007,
+        "details": [
+          "raw"
+        ],
+        "subsets": [],
+        "availability": "private",
+        "format": "packet",
+        "generation": "captured",
+        "covered_period": "months"
+      },
+      {
+        "name": "DARPA IDeval",
+        "availability": "public",
+        "format": "packet",
+        "types": [],
+        "generation": "altered_captured",
+        "generation_year": 1999,
+        "covered_period": "weeks",
+        "details": [
+          "raw"
+        ],
+        "subsets": []
+      }
+    ]
+  },
+  "preprocessing": {
+    "performed_feature_selection": true,
+    "packet_analysis_oriented": false,
+    "flow_analysis_oriented": true,
+    "flow_aggregation_analysis_oriented": false,
+    "tools": [],
+    "normalization_type": "no",
+    "transformations": [
+      "_histograms",
+      "_metric_space",
+      "entropy"
+    ],
+    "final_data_format": "_histograms",
+    "feature_selections": [
+      {
+        "name": "constant-removal",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      },
+      {
+        "name": "pca",
+        "type": "feature_reduction",
+        "classifier": "none",
+        "role": "main"
+      }
+    ],
+    "flows": [
+      {
+        "selection": "study_based",
+        "role": "main",
+        "main_goal": "anomaly_detection",
+        "active_timeout": "missing",
```

Comment:
Although they don't call it active timeout, they mention it is 5 minutes (300 seconds): "All histograms correspond to five
minute traces collected at the same time of the day."

Reply:
Ah, right. I think it's highly likely that the active time-out they use in their tool is larger than 300s, which would make the effective active time-out 300s. However, since they don't mention it, it's probably best to leave it as missing.